### PR TITLE
Write unit tests

### PR DESCRIPTION
**Pull request by rica213**
In this milestone, here are what we have done:
- Write unit tests to all the classes : Books, Person, Teacher, Student, Classroom, and Rental
- Update readme file accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Built on Ruby, this application streamlines the library management process for s
 - **Add basic UI.**
 - **Follow best practices.**
 - **Preserve data.**
+- **Implement unit tests.**
```
Comment (TedLivist) [inline README.md:53]: - [ ] Kindly add tests for the capitalize and trimmer decorator classes.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @rica213 and @shaaibu7,

Good job so far!
✔️ All tests are passing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @rica213, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rica213** [comment]
@TedLivist , @zieeco 

Thank you for taking the time in reviewing our project. We are grateful for your hard work, and your insights on how to improve our code. 🙇🏼‍♀️

**Pull request merged**